README.md

```diff
@@ -78,7 +78,9 @@ export default {
 | disableTextLayer       | `boolean`              | `true` or `false`                                | whether the text layer should be disabled                                  |
 | height                 | `number` <br> `string` | natural numbers                                  | desired page height in pixels (ignored if the width property is specified) |
 | page                   | `number`               | `1` to the last page number                      | number of the page to display (displays all pages if not specified)        |
-| source                 | `string` <br> `object` | document URL or typed array pre-filled with data | source of the document to display                                          |
+| source                 | `string` <br>          | document URL or typed array pre-filled with data | source of the document to display                                          |
+|                        | `object` <br>          |                                                  |                                                                            |
+|                        | `Uint8Array`           |                                                  |                                                                            |
```

Comment:
Ok, but let's put these types in one cell

```suggestion
| source                 | `string` <br> `object` <br> `Uint8Array` | document URL or typed array pre-filled with data | source of the document to display                                          |
```

Reply:
Ok! I saw that you had taken the trouble to get the source table cells aligned, and I tried to do the same. But, that's a losing battle in the end. :-)